### ISSUE TITLE
shiftzilla build returns an error on verison 0.2.22 

### ISSUE THREAD

**Issue by DirectedSoul1** [nhr/shiftzilla#12]
After following the installation link : 
https://github.com/nhr/shiftzilla/

```shiftzilla load``` and ```shiftzilla purge``` works fine but when ```shiftzilla build``` is issued
returns with an error as shown below 
```[root@localhost .shiftzilla]# shiftzilla triage
Traceback (most recent call last):
	16: from /usr/local/rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `<main>'
	15: from /usr/local/rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `eval'
	14: from /usr/local/rvm/gems/ruby-2.5.3/bin/shiftzilla:23:in `<main>'
	13: from /usr/local/rvm/gems/ruby-2.5.3/bin/shiftzilla:23:in `load'
	12: from /usr/local/rvm/gems/ruby-2.5.3/gems/shiftzilla-0.2.22/bin/shiftzilla:127:in `<top (required)>'
	11: from /usr/local/rvm/gems/ruby-2.5.3/gems/shiftzilla-0.2.22/lib/shiftzilla/engine.rb:83:in `triage_report'
	10: from /usr/local/rvm/gems/ruby-2.5.3/gems/shiftzilla-0.2.22/lib/shiftzilla/org_data.rb:22:in `populate_releases'
	 9: from /usr/local/rvm/gems/ruby-2.5.3/gems/shiftzilla-0.2.22/lib/shiftzilla/org_data.rb:22:in `each'
	 8: from /usr/local/rvm/gems/ruby-2.5.3/gems/shiftzilla-0.2.22/lib/shiftzilla/org_data.rb:28:in `block in populate_releases'
	 7: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:137:in `execute'
	 6: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:95:in `prepare'
	 5: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:143:in `block in execute'
	 4: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/statement.rb:107:in `each'
	 3: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/statement.rb:107:in `loop'
	 2: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/statement.rb:110:in `block in each'
	 1: from /usr/local/rvm/gems/ruby-2.5.3/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:147:in `block (2 levels) in execute'
/usr/local/rvm/gems/ruby-2.5.3/gems/shiftzilla-0.2.22/lib/shiftzilla/org_data.rb:30:in `block (2 levels) in populate_releases': undefined method `strip' for nil:NilClass (NoMethodError)```

The correct working scenario is when ```shiftzilla build``` is issued the logs in the form of html should be stored on the path defined in the shiftzilla_cfg.yaml file.

**Comment by nhr** [comment]
This issue was address with https://github.com/nhr/shiftzilla/pull/8 but you will need to upgrade to a newer Shiftzilla gem to get it. In short, if your BZ query comes back empty or malformed, you will see the error that you are reporting.

**Comment by DirectedSoul1** [comment]
This is solved with 0.2.23 gem , Thanks
